### DISCUSSION TITLE
zh/modes/benchmark/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#9688] [Docs]
# zh/modes/benchmark/

了解如何剖析YOLOv8 在各种导出格式中的速度和准确性；深入了解 mAP50-95、准确性_top5 指标等。

https://docs.ultralytics.com/zh/modes/benchmark/

**Comment by AA278-HUB** [comment]
it seems benchmark only run on Linux System ,It couldn't run on Window Enviroment?

**Reply by glenn-jocher** [reply]
It runs on Windows too—only some export formats are OS‑specific (e.g., CoreML/TF.js/EdgeTPU/RKNN aren’t supported on Windows); try a single supported format to verify your setup:
```bash
yolo benchmark model=yolo11n.pt data=coco8.yaml format=onnx device=cpu
```
If it still fails, please paste the `yolo checks` output and the full traceback here; format/OS notes are summarized in the benchmarks reference at [ultralytics.utils.benchmarks](https://docs.ultralytics.com/reference/utils/benchmarks/).

**Reply by AA278-HUB** [reply]
thank you,Very appreciate to your help .

**Reply by glenn-jocher** [reply]
Glad it helped! On Windows you can also benchmark a fast CPU path with OpenVINO:
```bash
yolo benchmark model=yolo11n.pt data=coco8.yaml format=openvino
```
Setup and tips are in the [OpenVINO export guide](https://docs.ultralytics.com/integrations/openvino/).
